### PR TITLE
Conceptual docs on replication

### PR DESCRIPTION
**Pull request by roji**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "numrange",
         "postgis",
         "postgres",
+        "postgresql",
```
Comment (YohDeadfall) [inline .vscode/settings.json:28]: Indentation.

Comment (roji) [reply .vscode/settings.json:28]: What exactly is the problem?

Comment (YohDeadfall) [reply .vscode/settings.json:28]: Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine.

Comment (YohDeadfall) [reply .vscode/settings.json:28]: Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine.

Comment (YohDeadfall) [reply .vscode/settings.json:28]: Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine. Plus it doesn't allow to reply you here.

Comment (roji) [reply .vscode/settings.json:28]: OK

**Pull request merged**